lib/lita_versioner/handlers/bumpbot_handler.rb

```diff
@@ -1,6 +1,8 @@
+require "lita"
 require "forwardable"
+require_relative "../jenkins_http"
 
-module Lita
+module LitaVersioner
```

Comment:
https://docs.lita.io/plugin-authoring/handlers/ recommends leaving handlers in Lita::Handlers.

At the very least, you'll need to change BumpbotHandler < Handler below to explicitly inherit from Lita::Handlers::Handler.